tools/slurm/scripts/slurmcluster.sh

```diff
@@ -76,7 +76,7 @@ while [[ $# -gt 0 ]]; do
             echo '  -A '
```

Comment:
I think the ```-A``` flag is the one we want to remove

Do we need any code changes to the scripts here to no longer support ```-A``` beyond changing the usage description?